docs/.vitepress/config/en.ts

```diff
@@ -64,7 +64,7 @@ function sidebarGeneral(): DefaultTheme.SidebarItem[] {
         //   link: "/general/setup-guide",
         // },
         {
-          text: "Emulator Support and Issues",
+          text: "Emulator Support",
           link: "/general/emulator-support-and-issues",
```

Comment:
Should the actual page be renamed too? Is there a redirector?

Reply:
The redirector lives at the nginx level and is configured in Forge.

I considered this, but it's probably more risk than it's worth, given how high-volume the traffic on the page is.